independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/handlers/ClientSendRequestHandler.java

```diff
@@ -88,30 +85,7 @@ public void handle(RestClientRequestContext requestContext) {
             return;
         }
         requestContext.suspend();
-        Uni<HttpClientRequest> future = createRequest(requestContext)
-                .runSubscriptionOn(new Executor() {
-                    @Override
-                    public void execute(Runnable command) {
-                        Context current = Vertx.currentContext();
-                        ClientRequestContextImpl clientRequestContext = requestContext.getClientRequestContext();
-                        Context captured = null;
-                        if (clientRequestContext != null) {
-                            captured = clientRequestContext.getContext();
-                        }
-                        if (current == captured || captured == null) {
-                            // No need to switch to another context.
-                            command.run();
-                        } else {
-                            // Switch back to the captured context
-                            captured.runOnContext(new Handler<Void>() {
-                                @Override
-                                public void handle(Void ignored) {
-                                    command.run();
-                                }
-                            });
-                        }
-                    }
-                });
+        Uni<HttpClientRequest> future = createRequest(requestContext);
 
         // DNS failures happen before we send the request
```

Comment:
Do you know why we have this comment? Someone recently reported a DNS resolution failure when not on a duplicated context.

Reply:
It was added as part of the MicroProfile 4 and Vert.x 4 upgrade a while ago: https://github.com/quarkusio/quarkus/pull/16333

Commit: https://github.com/quarkusio/quarkus/pull/16333/commits/0b88e3cc99e131a565de1a02aa8c819adce44fac